source/frontend/src/layouts/AuthLayout.tsx

```diff
@@ -35,7 +35,6 @@ const Content = styled.div`
   grid-area: content;
   overflow-y: hidden;
   overflow-x: auto;
-  min-width: 154rem;
```

Comment:
This was forcing the screen width to be 1540px - and displaying a scroll bar if less. Given all of the changes in this PR, this is no longer necessary.